### ISSUE TITLE
Documentation on flexible extensions ?

### ISSUE THREAD

**Issue by Depado** [russross/blackfriday#396]
Hi 👋 

First of all, I just love this project, so congratulations to the people that made it possible 🎉 

In the README of the v2 version it is said :
> Flexibility to easily add your own rendering extensions

Is there any documentation on how to achieve this other than diving in the code ?
Or if anyone already did something like that, I'd love an example

Thanks a lot 🎆

**Comment by rtfb** [comment]
For a super quick answer, here are a couple of gists demonstrating custom rendering:
https://gist.github.com/rtfb/f0ad5013f4c3068b592da205b6542db9
https://gist.github.com/rtfb/2693f6bfcc1760661e8d2fb832763a15

But a note taken: we need more examples like above, in an place that's easy to find.

Glad you like it! ;-)

**Comment by Depado** [comment]
Awesome ! Thanks a lot !
If I can help elaborating the documentation once I understand how all of this work, just tell me where I should do that 😄

**Comment by rtfb** [comment]
Get back on this issue when you have something, and we'll figure out! :-)

**Comment by Depado** [comment]
https://gist.github.com/Depado/359e425bb7b188fd2f642eebd23b9222
Am I doing that right ? 😄

**Comment by rtfb** [comment]
Yup, you're doing it right. You can actually go all the way and make it a package with an exported renderer. Then people will be able to import both Blackfriday and your renderer, and use them together. That would be super neat! ;-)

**Comment by Depado** [comment]
I'll do that as soon as I get some time !
Thanks for your help 👍

**Comment by rtfb** [comment]
When you do it, I think it will qualify as a fix for the oldest outstanding Blackfriday issue, #10 ;-)

**Comment by Depado** [comment]
https://github.com/Depado/bfchroma
How does that sound ? Anything I should add or anything that is not right ?

**Comment by rtfb** [comment]
Neat! I have added a couple comments over there: https://github.com/Depado/bfchroma/issues/1

**Comment by Depado** [comment]
Thanks for your issue that's awesome :)

**Comment by amkgo** [comment]
Is it possible to add customer container support like [markdown-it-container](https://github.com/markdown-it/markdown-it-container)?

**Comment by Depado** [comment]
I don't know if that's possible but that would be awesome to integrate that
